conf/application.conf

```diff
@@ -165,5 +166,31 @@ microservice {
             host=localhost
             port=8585
         }
+
+        service-locator {
+            host=localhost
+            port=9602
+        }
+    }
+}
+
+Prod {
```

Comment:
Is there a need for the Prod configuration?
I think usually the norm for code in the open is to have only microservice.services.service-locator and then override it in https://github.tools.tax.service.gov.uk/HMRC/app-config-base/blob/master/api-platform-test-user.conf#L33
Possibly to avoid having configuration exposed in the open?